tensorflow/python/estimator/inputs/queues/feeding_functions_test.py

```diff
@@ -290,6 +290,40 @@ def testOrderedDictNumpyFeedFnLargeBatchWithSmallArrayAndMultipleEpochs(self):
     actual = aff()
     self.assertEqual(expected, vals_to_list(actual))
 
+  def testFillArraySmall(self):
+    a = np.ones(shape=[32, 32], dtype=np.int32).tolist() + \
+          np.ones(shape=[32, 36], dtype=np.int32).tolist()
+    actual = np.ones(shape=[64, 36], dtype=np.int32)
+    ff._fill_array(actual, a)
+    expected = np.ones(shape=[64, 36], dtype=np.int32)
+    expected[:32, 32:] = 0
```

Comment:
Can you test (either here or even better in a separate test case) that a value other than 0 is actually used if specified?